pythonmod/pythonmod.c

```diff
@@ -3,6 +3,7 @@
  *
  * Copyright (c) 2009, Zdenek Vasicek (vasicek AT fit.vutbr.cz)
  *                     Marek Vavrusa  (xvavru00 AT stud.fit.vutbr.cz)
+ * Copyright (c) 2023, Rubicon Communications, LLC (Netgate)
```

Comment:
Not sure about the copyright note here for this small change.
Usually we don't do copyrights for fixes. If there is copyright in Unbound it is mainly from the original authors, as in this case.
Is the copyright addition something necessary in your case?
Btw acknowledgment for contributions are always noted in commit/merge messages, but also in the Changelog (edited by the Unbound development team) and release notes.

Reply:
> Not sure about the copyright note here for this small change. Usually we don't do copyrights for fixes. If there is copyright in Unbound it is mainly from the original authors, as in this case. Is the copyright addition something necessary in your case? Btw acknowledgment for contributions are always noted in commit/merge messages, but also in the Changelog (edited by the Unbound development team) and release notes.

Not necessary. Removed. Acknowledgement is fine in the commit/merge and Changelog. Thanks